Podfile.lock

```diff
@@ -2,9 +2,9 @@ PODS:
   - Alamofire (4.7.3)
   - AlamofireImage (3.4.1):
     - Alamofire (~> 4.7)
-  - Apollo (0.9.3):
-    - Apollo/Core (= 0.9.3)
-  - Apollo/Core (0.9.3)
+  - Apollo (0.10.1):
```

Comment:
Should we be updating these pods?

Reply:
Using version 0.9.3 of Apollo gives me a "Cannot assign value of type 'EnumeratedSequence<[Key]>.Iterator' to type 'EnumeratedSequence<IndexingIterator<Array<Key>>>.Iterator", which I found out can be caused by Apollo. Updated the pod fixed the issue.

Reply:
Ahh good. Glad they finally updated the version.